website/frontend/src/pages/AboutUsPage.js

```diff
@@ -31,29 +31,30 @@ import UNEPLogo from 'assets/img/partners/UNEP.png';
 import UniversityOfColoradoBoulderLogo from 'assets/img/partners/University of Colorado Boulder US.png';
 import UniversityOfFloridaLogo from 'assets/img/partners/University of Florida US.png';
 import USEmbassyKampalaLogo from 'assets/img/partners/US Embassy UG.png';
+import TAHMO from 'assets/img/partners/TAHMO.png';
+import MMU from 'assets/img/partners/MMU.jpg'
```

Comment:
new logos

Reply:
Brilliant @FaithDaka , thanks! I hope we have paid some attention to file names to prevent the other file system issue from happening again.

Reply:
In the future, we shall need to have a **naming conventions** for logos just to minimise any file system... cc: @Codebmk